### PR TITLE
PHOENIX-7080 Switch phoenix-thirdparty to guava-jre from guava-android

### PR DESCRIPTION
**Pull request by NihalJain**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -68,7 +68,7 @@
 
   <properties>
     <rename.offset>org.apache.phoenix.thirdparty</rename.offset>
-    <guava.version>32.1.1-android</guava.version>
+    <guava.version>32.1.2-jre</guava.version>
```
Comment (NihalJain) [inline pom.xml]: 32.1.3-jre released just last week. let me know if should bump

Comment (stoty) [reply pom.xml]: I see no reason not to use the latest.

Comment (NihalJain) [reply pom.xml]: Updated!

**Review by stoty — APPROVED** [review]
+1 LGTM.

Please do run a full Phoenix test suite with this change in a snapshot version before committing.

**Comment by NihalJain** [comment]
Ran tests for phoenix 5.1 branch locally with this change. No failures seen due to guava change. Please let me know if anything else if needed here.

**Comment by stoty** [comment]
Thank you.

**Pull request closed without merging**